### PR TITLE
37 build archetypes

### PR DESCRIPTION
**Pull request by kmcelwee**
* It's safe to say we'll always have an `images` sub-directory? This requires a `.keep` file that we'll have to delete.
* Should I delete `archetypes/default.md` and `themes/startwords/archetypes/default.md`? I don't think they're adding anything and might be from when we were first messing around with this.

### CODE REVIEW

**Review by rlskoeser — APPROVED** [review]
I don't think we should include the images subdirectory in the archetype if we don't have any archetypal content to put in it. I would just add to the readme notes that by convention we put images in a subdirectory. (Could even provide the `mkdir` command to generate it for the path of the article that was just created, although not necessary).

I'm going back and forth on whether documentation should be in the readme or in the templates. It seems like it makes the templates messier and more cluttered, but that's also where editors are guaranteed to be looking.

I don't think you should delete the default archetype — I think hugo added that for us when we created the new theme, recommend we keep.

I have a bunch of suggestions to revise wording and tweak the templates. In particular I do think using the current day's date would be more useful than setting it to 2100.  

You can merge this in when you're happy with the revisions, but Grant should test and sign off on using the new archetypes when he has time, so we make sure all the instructions and templates make sense to him.

File: themes/startwords/archetypes/article/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+type: article
+title: {{ replace .Name "-" " " | title }}
+slug: {{ .Name }}
+order: 1 # Change me
+authors:
+  - INSERT
+  - AUTHORS
+date: 2120-10-01 # Change me
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/archetypes/article/index.md]: 1a70cb1c1f4a1c6e5875454c0619a844ecf78296  ✅

File: themes/startwords/archetypes/article/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+images: ["{{ .File.Dir }}images/social-media-preview.jpg"]
+summary: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/archetypes/article/index.md]: afbc6ff42f774b454bbb8039fa5f4241e63b32df ✅

File: themes/startwords/README.md [hunk 3]
```diff
@@ -58,6 +58,22 @@ Editors working on publishing new articles should review the text versions of th
...
 
+## Generate new issues and articles
+
+You can use [hugo archetypes](https://gohugo.io/content-management/archetypes/#directory-based-archetypes) to quickly build new issues and articles. To build the fourth issue, for example, use this command and then edit the metadata:
+
+```sh
+hugo new --kind issue issues/4
+```
+
+And to publish a new article named "A Cup of Tea":
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/README.md]: 0a0e3e2e636981463c0c372990a42e79e453a14a ✅

File: themes/startwords/README.md [hunk 4]
```diff
@@ -58,6 +58,22 @@ Editors working on publishing new articles should review the text versions of th
...
+
+```sh
+hugo new --kind issue issues/4/a-cup-of-tea
+```
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/README.md:77]: 0a0e3e2e636981463c0c372990a42e79e453a14a ✅

File: themes/startwords/README.md [hunk 5]
```diff
@@ -58,6 +58,22 @@ Editors working on publishing new articles should review the text versions of th
...
+
+Both of these commands need to be performed at the root directory.
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/README.md]: 0a0e3e2e636981463c0c372990a42e79e453a14a ✅

File: themes/startwords/README.md [hunk 6]
```diff
@@ -58,6 +58,22 @@ Editors working on publishing new articles should review the text versions of th
...
 The regular expressions used to convert article markdown to the plain text article display are managed in a [data file](data/article_txt_replace.toml), with comments to document the purpose of each regular expression.
 
+## Generate new issues and articles
+
+You can use [hugo archetypes](https://gohugo.io/content-management/archetypes/#directory-based-archetypes) to quickly build new issues and articles. To build the fourth issue, for example, use this command and then edit the metadata:
+
+```sh
+hugo new --kind issue issues/4
+```
+
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/README.md:68]: 0a0e3e2e636981463c0c372990a42e79e453a14a ✅

File: themes/startwords/archetypes/issue/_index.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+---
+type: issue
+layout: single
+title: Issue {{ .Name }}
+number: {{ .Name }}
+theme: INSERT_THEME
+# Unless the publish date is before today's date, hugo won't publish it.
+date: 2100-12-01 
+slug: {{ .Name }}
+summary: Insert the beginning of the editor's letter
```
[the remark this answers is not in the document]
Comment (kmcelwee) [reply themes/startwords/archetypes/issue/_index.md]: abedfae2c85bdeb0ea8b0dcb930a672fcf35f594 ✅

**Comment by kmcelwee** [comment]
> I don't think we should include the images subdirectory in the archetype if we don't have any archetypal content to put in it. I would just add to the readme notes that by convention we put images in a subdirectory. (Could even provide the mkdir command to generate it for the path of the article that was just created, although not necessary).

97cbaf3 ✅

> I'm going back and forth on whether documentation should be in the readme or in the templates. It seems like it makes the templates messier and more cluttered, but that's also where editors are guaranteed to be looking.

I ended up putting some of it in both. I agree that we should prioritize information in the place where an editor is looking. And we'll get feedback from Grant with any confusion I guess.

**Pull request merged**